assets/js/theme/global/currency-selector.js

```diff
@@ -1,5 +1,57 @@
-export default function () {
+import swal from './sweet-alert';
+import utils from '@bigcommerce/stencil-utils';
+
+export default function (cartId) {
+    function changeCurrency(url, currencyCode) {
+        $.ajax({
+            url,
+            method: 'POST',
+            data: JSON.stringify({ currencyCode }),
+        }).then(() => {
+            window.location.reload();
+        });
```

Comment:
We should probably add a `catch` here and display an error modal if something goes wrong in the request.